app/views/layouts/_header.html.erb

```diff
@@ -43,12 +43,12 @@
               <a class="dropdown-item" href="/getting-started"><b><%= translation('layout._header.community.get_started') %> &raquo;</b></a>
             <% end %>
               <div class="dropdown-divider"></div>
-              <%= create_nav_dropdown_item("people","layout._header.community.people") %>
-              <%= create_nav_dropdown_item("map","layout._header.community.places") %>
-              <%= create_nav_dropdown_item("projects","layout._header.community.projects") %>
-              <%= create_nav_dropdown_item("events","layout._header.community.events") %>
-              <%= create_nav_dropdown_item("questions","layout._header.community.questions") %>
-              <%= create_nav_dropdown_item("chat","layout._header.community.chat") %>
+              <%= create_nav_dropdown_item("/people","layout._header.community.people") %>
```

Comment:
and on every instance of the use of this helper. Should work!

Reply:
Aah yes, I got it. Thanks for pointing this out! This is a much elegant way.

I guess then for the external site URLs, we can use the https:// URLs now, right? Since it won't cause a problem now, and having the // in the URL seems improper.